src/wallet/sqlite.cpp

```diff
@@ -34,6 +34,10 @@ SQLiteDatabase::SQLiteDatabase(const fs::path& dir_path, const fs::path& file_pa
     LogPrintf("Using SQLite Version %s\n", SQLiteDatabaseVersion());
     LogPrintf("Using wallet %s\n", m_dir_path);
 
+    int ret = sqlite3_initialize();
```

Comment:
In commit "Initialize and Shutdown sqlite3 globals" (5c78c9094a0158e4d466b696a3bbbcbd35b51b15)

Minor: Would add comment saying this is a no-op if it is already called. Another option would be to only call it when g_file_paths is empty.

Reply:
Done